src/testing/mocking.md

```diff
@@ -0,0 +1,44 @@
+---
+minutes: 10
+---
+
+# Mocking
+
+If you want to do mocking, we recommend [Mockall]. You need to refactor your
+code to use traits, which you can then quickly mock:
```

Comment:
We should first mention that we generally recommend against mocking, as per go/choose-test-double and go/mocking.

Reply:
Yeah, but not in the public documentation... I added the "if" because of that, but let me make it stronger by adding a note to the speaker notes! Thanks for the idea :smile: